operators/grpc_operators/python/client/grpc_client_request.py

```diff
@@ -0,0 +1,53 @@
+# SPDX-FileCopyrightText: Copyright (c) 2025 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import asyncio
+import logging
+
+from holoscan.core import Operator, OperatorSpec
+
+from operators.grpc_operators.python.common.tensor_proto import TensorProto
+
+
+class GrpcClientRequestOp(Operator):
+    @property
+    def cuda_stream_pool(self):
+        return self.cuda_stream_pool
```

Comment:
should return `self._cuda_stream_pool` to avoid recursion, but can just remove the method as unused?